.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+## Issue Details
+(closes, fixes, suggestion) for #issue-number 
```

Comment:
We probably want to add a few comments as to what exactly does closes, fixes, suggestions means and what it does. Since we will be having contributors who are new to git and github and would not know what these terms mean or do.

Reply:
Yes. making it more inclusive. I have added comments.